test/libsolidity/smtCheckerTests/types/string_1.sol

```diff
@@ -1,10 +1,10 @@
-pragma experimental SMTChecker;
-
 contract C
 {
 	function f(string memory s1, string memory s2) public pure {
 		assert(bytes(s1).length == bytes(s2).length);
 	}
 }
+// ====
+// SMTEngine: all
 // ----
-// Warning 6328: (110-154): CHC: Assertion violation happens here.\nCounterexample:\n\ns1 = [26, 26, 26, 26, 26, 26, 26, 9, 26, 26, 26, 26, 26, 14, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26]\ns2 = [18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 15, 18, 18, 18, 18, 18, 22, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18]\n\nTransaction trace:\nC.constructor()\nC.f([26, 26, 26, 26, 26, 26, 26, 9, 26, 26, 26, 26, 26, 14, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26, 26], [18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 15, 18, 18, 18, 18, 18, 22, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18, 18])
+// Warning 6328: (77-121): CHC: Assertion violation happens here.\nCounterexample:\n\n\nTransaction trace:\nC.constructor()\nC.f(s1, s2)
```

Comment:
`s1` and `s2` are not defined in the new counterexample. Is this valid?

Reply:
Yea, it can happen that the value is given in a complicated way which we can't parse, so we just don't output them in smtlib form